build/tutorials/tutorials-contest/pangolin-token-subgraph/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
No need to add a .gitignore file since we won't be having people install the app in this directory.

Reply:
Deleted .gitignore